drivers/bus/acpi_new/acpi.rc

```diff
@@ -0,0 +1,5 @@
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "uACPI ReactOS ACPI Driver"
+#define REACTOS_STR_INTERNAL_NAME     "uACPI"
+#define REACTOS_STR_ORIGINAL_FILENAME "uACPI.sys"
```

Comment:
Please keep support for the original file name. This allows an application to determine whether the file has been renamed.
```suggestion
#define REACTOS_STR_FILE_DESCRIPTION  "uACPI ReactOS ACPI Driver"
#define REACTOS_STR_INTERNAL_NAME     "uACPI"
#define REACTOS_STR_ORIGINAL_FILENAME "acpi.sys"
```

Reply:
oh, okay then lol ill do it i guess